examples/webcomponents/index.html

```diff
@@ -47,6 +51,14 @@
             ReactDOM.render(<a href={url}>{name}</a>, mountPoint);
           }
         }
+        ,detachedCallback: {
+          value: function() {
+          }
+        }
+        ,attributeChangedCallback: {
```

Comment:
Afaik, we don't need to define `createdCallback`, `detachedCallback`, nor `attributeChangedCallback`.  It just makes the example more complicated and detracts from what we're demonstrating.  Can you remove these additional functions?